src/cli.cr

```diff
@@ -11,14 +11,20 @@ module DppmRestApi::CLI
       port ||= dppm_config.port
       host ||= dppm_config.host
     end
-    DppmRestApi.run port, host
+    DppmRestApi.run host, port, data_dir
```

Comment:
The other options are already set by DPPM?

Reply:
Yes, this variables comes from the CLI, which has default values (below).